replica/replica.go

```diff
@@ -163,6 +163,9 @@ func construct(readonly bool, size, sectorSize int64, dir, head string, backingF
 		logrus.Errorf("failed to create directory: %s", dir)
 		return nil, err
 	}
+	if err := findExtents(dir); err != nil {
```

Comment:
This should be checked just once while starting replica but Construct function is called multiple times.
Lets also give a better name for this function, like isExtentsSupported, or something else.